WcaOnRails/spec/models/delegate_report_spec.rb

```diff
@@ -34,6 +34,22 @@
     expect(dr.discussion_url).to eq "https://groups.google.com/a/worldcubeassociation.org/forum/#!topicsearchin/reports/" + URI.encode_www_form_component(dr.competition.name)
   end
 
+  it "wrc_feedback_requested is set on creation" do
```

Comment:
I was hoping to verify that it was `false` by default. Might be overkill, but my reasoning was that I wanted it to be 'set' (as in not null) by default, and I didn't want it to be `true` (since a `null` wrc_incidents field + `true` wrc_feedback_requested would cause problems). Do you have any recommendations for a cleaner/clearer way of testing something like this?

Reply:
Got it. So this is kind of testing the database migration you wrote? That seems fine to me. I think there's some ambiguity about what "set" means here, it might be clearer to rename this test to "... is set false on creation".

Reply:
Yep! Agreed. I've updated the name.